src/test/java/uk/gov/hmcts/reform/jobscheduler/controllers/jobscontroller/CreateTest.java

```diff
@@ -45,16 +46,24 @@ public void should_fill_location_header_on_successful_save() throws Exception {
         given(jobsService.create(any(Job.class), anyString()))
             .willReturn(id);
 
-        send(SampleData.validJobJson())
+        send(SampleData.jobJson())
             .andExpect(header().string(LOCATION, endsWith("/jobs/" + id)));
     }
 
+    @Test
+    public void should_validate_model() throws Exception {
+        String jobWithoutUrl = SampleData.jobJson("");
+
+        send(jobWithoutUrl)
+            .andExpect(status().isBadRequest());
```

Comment:
What's the response here? Is it just a bad request or is there a body indicating what's wrong? I'm thinking about covering other fields.

Reply:
currently response body is empty